user/widle/template.py

```diff
@@ -0,0 +1,22 @@
+pkgname = "widle"
+pkgver = "1.0"
+pkgrel = 0
+build_style = "makefile"
+hostmakedepends = [
+    "pkgconf",
```

Comment:
for wayland-scanner, pulled in by wayland-devel but that's in makedepends so not strictly correct
```suggestion
    "pkgconf",
    "wayland-progs"
```